admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/toolingpage.html

```diff
@@ -111,79 +111,90 @@
+            loadComponent('admin-components-spinner')
+            loadComponent('admin-components-tour')
+            loadComponent('admin-components-about')
+            loadComponent('admin-components-explorerpreview')
+            loadComponent('admin-components-explorerpreviewnavitem')
+            loadComponent('admin-components-explorerpreviewcontent')
+            loadComponent('admin-components-confirmdialog')
+            loadComponent('admin-components-fileupload')
+            loadComponent('admin-components-separator')
+            loadComponent('admin-components-colorpaletteselector')
+            loadComponent('admin-components-materializeswitch')
+            loadComponent('admin-components-componentexplorer')
+            loadComponent('admin-components-nodetreeitem')
+            loadComponent('admin-components-publishingmodal')
+            loadComponent('admin-components-publishinginfo')
+            loadComponent('admin-components-richtoolbar')
+            loadComponent('admin-components-richtoolbargroup')
+            loadComponent('admin-components-richtoolbarbtn')
+            loadComponent('admin-components-addcomponentmodal')
+            loadComponent('admin-components-icon')
+            loadComponent('admin-components-tabswrapper')
+            loadComponent('admin-components-tab')
+
+
+            loadComponent('admin-components-materializedropdown')
+            loadComponent('admin-components-dropdowndivider')
+            loadComponent('admin-components-materializemodal')
+            loadComponent('admin-components-materializespinner')
+
+            loadComponent('field-pathbrowser')
+            loadComponent('field-texteditor')
+            loadComponent('field-list')
+            loadComponent('field-collection')
+            loadComponent('field-materialswitch')
+            loadComponent('field-material-checkbox')
+            loadComponent('field-material-radios')
+            loadComponent('field-material-select')
+            loadComponent('field-material-checklist')
+            loadComponent('field-material-datepicker')
+            loadComponent('field-material-timepicker')
+            loadComponent('field-material-datetime')
+            loadComponent('field-material-textarea')
+            loadComponent('field-material-multiselect')
+            loadComponent('field-material-range')
+            loadComponent('field-listselection')
+            loadComponent('field-iconbrowser')
+            loadComponent('field-headline')
+            loadComponent('field-horizontal-line')
+
+            loadComponent('admin-components-iconeditpage')
+            loadComponent('admin-components-iconopenfolder')
+            loadComponent('admin-components-iconrename')
+            loadComponent('admin-components-draghandle')
+            loadComponent('admin-components-iconbrowser')
+            loadComponent('pagerendervue-components-placeholder')
+        })($perAdminApp.loadComponent)
```

Comment:
- I like this approach but it could be done even better
  - Make and array of all the preloaded components and then simply call
  ```javascript
  const preload = [
    'admin-components-iconaction',
    //...
  ]
  preload.forEach((componentName) => $perAdminApp.loadComponent(componentName))
  ```
- or even better:
  - if i recall correct you can use the vue-lifecycle to load the lazy by using the `beforeCreate()` hook
  - **BUT** that would take alot of changes